src/customer.js

```diff
@@ -1,9 +1,22 @@
+import Hotel from './hotel'
 class Customer {
   constructor(customer) {
     this.id = customer.id;
     this.name = customer.name;
     this.bookings = [];
+    this.expenses = null;
   }
+
+  setBookings(hotel) {
+    this.bookings = hotel.getFullRoomInfoForBookings(this)
+    return this.bookings
+  }
+
+  getExpenses(hotel) {
+    this.expenses = hotel.getUserExpenses(this) 
+    return this.expenses
+  }
```

Comment:
It looks like the jobs of these methods are to call upon methods from the Hotel class and return their results - is there a way to simply call the methods directly from Hotel instead?   

Unless you are using ‘this.bookings’ and ‘this.expenses’ in other places within Customer, which I could see the benefit of holding onto this info.